src/main/java/igrad/model/module/Module.java

```diff
@@ -24,6 +25,7 @@
     private final Memo memo;
     private final Description description;
```

Comment:
@waynewee might be a good idea to change all the optional fields to optional too, e.g, Description, since Grade is already Optional